tensorflow/compiler/tests/image_ops_test.py

```diff
@@ -91,13 +91,13 @@ def testRGBToHSVNumpy(self):
     """Tests the RGB to HSV conversion matches a reference implementation."""
     for nptype in self.float_types:
       rgb_flat = _generate_numpy_random_rgb((64, 3)).astype(nptype)
-      rgb_np = rgb_flat.reshape(4, 4, 4, 3)
+      rgb_np = rgb_flat.reshape(4, 4, 4, 3)  # pylint: disable=too-many-function-args
       hsv_np = np.array([
           colorsys.rgb_to_hsv(
               r.astype(np.float64), g.astype(np.float64), b.astype(np.float64))
           for r, g, b in rgb_flat
       ])
-      hsv_np = hsv_np.reshape(4, 4, 4, 3)
+      hsv_np = hsv_np.reshape(4, 4, 4, 3)  # pylint: disable=too-many-function-args
```

Comment:
E.g. We are tracking this type of notifications with https://github.com/tensorflow/tensorflow/issues/41873